__tests__/app.test.js

```diff
@@ -61,14 +61,91 @@ describe("GET /api/reviews", () => {
         });
       });
   });
-  test("200: should respond with reviews ordered by date descending", () => {
+  test("200: should respond with reviews ordered by date descending by default", () => {
     return request(app)
       .get("/api/reviews")
       .expect(200)
       .then((result) => {
         expect(result.body.reviews).toBeSortedBy("created_at", { descending: true });
       });
   });
+  describe("Queries", () => {
+    describe("Sort_By", () => {
+      test("200: should sort by given column", () => {
+        return request(app)
+          .get("/api/reviews?sort_by=title")
+          .expect(200)
+          .then((result) => {
+            expect(result.body.reviews).toBeSortedBy("title", { descending: true });
+          });
+      });
+      test("400: should reject columns that arent in the DB", () => {
+        return request(app)
+          .get("/api/reviews?sort_by=randomcolumnname")
+          .expect(400)
+          .then((result) => {
+            expect(result.body.msg).toBe("Bad Request");
+          });
+      });
+    });
+    describe("Order", () => {
+      test("200: should order by ascending", () => {
+        return request(app)
+          .get("/api/reviews?order=asc")
+          .expect(200)
+          .then((result) => {
+            expect(result.body.reviews).toBeSortedBy("created_at");
+          });
+      });
+      test("200: should order by descending", () => {
+        return request(app)
+          .get("/api/reviews?order=desc")
+          .expect(200)
+          .then((result) => {
+            expect(result.body.reviews).toBeSortedBy("created_at", { descending: true });
+          });
+      });
+      test("200: should default to descending given an invalid argument", () => {
+        return request(app)
+          .get("/api/reviews?order=AnyOrderPlease")
+          .expect(200)
+          .then((result) => {
+            expect(result.body.reviews).toBeSortedBy("created_at", { descending: true });
+          });
+      });
+    });
+    describe("Category", () => {
+      test("200: should return reviews with a given category", () => {
+        return request(app)
+          .get("/api/reviews?category=euro game")
+          .expect(200)
+          .then((result) => {
+            expect(result.length > 0);
```

Comment:
This assertion isn't really doing what you want 

- Firstly, you want to make sure you are checking the length of the `result.body.reviews`, since that is the value that contains you want 
- Secondly, the assertion needs a `toBe(true)` or something similar to actually perform the check!